### PR TITLE
Audit changes

### PR DESCRIPTION
**Pull request by gulshanvas**

### CODE REVIEW

**Review by ashwinYardi — DISMISSED** [review]

File: contracts/StakingPool.sol [hunk 1]
```diff
@@ -553,9 +569,9 @@ contract StakingPool is
     ) private {
         uint256 rewardBal = _rewardToken.balanceOf(address(this));
         if (_amount > rewardBal) {
-            _rewardToken.transfer(_to, rewardBal);
+            _safeTransfer(address(_rewardToken), _to, rewardBal);
```
Comment (ashwinYardi) [inline contracts/StakingPool.sol:572]: Can we use SafeERC20?

**Review by ashwinYardi — CHANGES_REQUESTED** [review]

File: contracts/StakingPool.sol [hunk 2]
```diff
@@ -541,6 +548,15 @@ contract StakingPool is
         rewardInfo.rewardToken.transfer(msg.sender, _amount);
     }
 
+    // Wrapper for safeTransfer
+    function _safeTransfer(
```
Comment (ashwinYardi) [inline contracts/StakingPool.sol:552]: Dont require this function now. :)

**Review by ashwinYardi — CHANGES_REQUESTED** [review]

File: contracts/StakingPool.sol (hunk 2 above)
Comment (gulshanvas) [reply contracts/StakingPool.sol]: Removed

File: contracts/StakingPool.sol (hunk 1 above)
Comment (gulshanvas) [reply contracts/StakingPool.sol]: Yes

**Review by gulshanvas — COMMENTED** [review]
PR is again available for review

**Review by ashwinYardi — APPROVED** [review]

**Pull request merged**